priv/repo/migrations/20200316103722_create_user_auth_tables.exs

```diff
@@ -0,0 +1,26 @@
+defmodule Demo.Repo.Migrations.CreateUserAuthTables do
```

Comment:
```suggestion
defmodule Demo.Repo.Migrations.CreateUsersAuthTables do
```
The filename should also be updated to `priv/repo/migrations/20200316103722_create_users_auth_tables.exs`